get-started/part4.md

```diff
@@ -67,35 +67,41 @@ machine a swarm manager. From then on, Docker will run the commands you execute
 on the swarm you're managing, rather than just on the current machine.
 
 {% capture local-instructions %}
-You now have two VMs created, named `myvm1` and `myvm2` (as `docker-machine ls`
-shows). The first one will act as the manager, which executes `docker` commands
+You now have two VMs created, named `myvm1` and `myvm2`:
+
+```shell
+docker-machine ls
+```
+
+The first one will act as the manager, which executes `docker` commands
 and authenticates workers to join the swarm, and the second will be a worker.
 
 You can send commands to your VMs using `docker-machine ssh`. Instruct `myvm1`
 to become a swarm manager with `docker swarm init` and you'll see output like
 this:
 
 ```shell
-$ docker-machine ssh myvm1 "docker swarm init"
+$ docker-machine ssh myvm1 "docker swarm init --advertise-addr <myvm1 ip>"
 Swarm initialized: current node <node ID> is now a manager.
 
 To add a worker to this swarm, run the following command:
 
   docker swarm join \
   --token <token> \
-  <ip>:<port>
+  <myvm ip>:<port>
```

Comment:
Again, we usually include the instructions verbatim. It would be acceptable to change the IP addresses to something in the documentation ranges like the IP address equivalents of example.com.